controllers/utils/policy_template.go

```diff
@@ -95,6 +103,17 @@ func VerifyHubTemplateFunctions(tmpl interface{}, policyName string) error {
 			if matches[7] != "" {
 				return &PolicyErr{matches[7], PlcHubTmplPrinfInNameErr}
 			}
+		} else if hubTmplFunc == "lookup" {
+			matches := regexpLookup.FindStringSubmatch(hubTmpl)
+
+			// Hub template doesn't match the regular expression
+			if len(matches) == 0 {
+				return &PolicyErr{hubTmpl, PlcHubTmplFmtErr}
+			}
+
+			if !strings.HasPrefix(matches[1], "cluster.open-cluster-management.io/") || matches[2] != "ManagedCluster" || matches[3] != "" {
```

Comment:
are these case-sensitive?

Reply:
Yes, they are case-sensitive. 
ACM reports the hub template error - one or more API resources are not installed when apiVersion/kind has incorrect capitalization.